recipe/meta.yaml

```diff
@@ -33,6 +35,7 @@ requirements:
     - pydata-google-auth
     - pymapd >=0.12.0
     - pymysql
+    - pyspark >=2.4.3
```

Comment:
Do tests fail without this? I don't think we should add this as a requirement, it's pretty heavyweight and requires a JVM installation on the host machine to work properly.

Reply:
I just add this to see if it will change the result .. nothing special I will remove that!